soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3717,12 +3716,11 @@ void DrawRandoEditor(bool& open) {
             return;
         }
 
-        bool disableEditingRandoSettings = CVar_GetS32("gRandoGenerating", 0) ||
-                                           CVar_GetS32("gOnFileSelectNameEntry", 0);
+        bool disableEditingRandoSettings = CVar_GetS32("gRandoGenerating", 0) || CVar_GetS32("gOnFileSelectNameEntry", 0);
+        bool disableZeldaRelatedOptions = CVar_GetS32("gRandomizeSkipChildZelda", 0);
```

Comment:
Now that this just always matches the cvar, we can probably skip making a new bool

Reply:
I think it did actually improve readability a little, but it's been changed now.